tests/deprecated_api/test_remove_1-4.py

```diff
@@ -130,16 +130,6 @@ def test_v1_4_0_deprecated_metrics():
     with pytest.deprecated_call(match='will be removed in v1.4'):
         precision_recall(torch.randint(0, 2, (10, 3, 3)), torch.randint(0, 2, (10, 3, 3)))
 
-    # Testing deprecation of class_reduction arg in the *new* precision
-    from pytorch_lightning.metrics.functional import precision
-    with pytest.deprecated_call(match='will be removed in v1.4'):
-        precision(torch.randint(0, 2, (10, )), torch.randint(0, 2, (10, )), class_reduction='micro')
-
-    # Testing deprecation of class_reduction arg in the *new* recall
-    from pytorch_lightning.metrics.functional import recall
-    with pytest.deprecated_call(match='will be removed in v1.4'):
-        recall(torch.randint(0, 2, (10, )), torch.randint(0, 2, (10, )), class_reduction='micro')
```

Comment:
is the only above scope, that with this routing we drop deprecated `class_reduction` already in 1.3, not 1.4
 but I feel it is fine as we simplify the codebase...

Reply:
i agree with this :]